environ.py

```diff
@@ -30,7 +30,7 @@
 LOG_LEVEL: str = ENV.get('LOG_LEVEL', 'INFO')
 logging.basicConfig(
     level=LOG_LEVEL,
-    format='%(asctime)s - %(name)s - %(funcName)s - %(levelname)s  - %(message)s'
+    format='%(asctime)s - %(name)s - %(funcName)s - %(levelname)s - %(message)s'
```

Comment:
I'd commented in #157 that I liked the log format that had been used at the time and I'd be happy with the addition of `funcName`. (Surprising appearance of camelCase, no?)  However, now that I've seen the log output, in this format, I like the delimiters with dash and spaces.  (Although, not **_too many_** spaces. 😉)

I also like the addition of the time.  I didn't think about it as I run the program locally, but having the timetstamp is pretty important for troubleshooting while running it on a server.  I don't really like the comma separator for fractional seconds, but it's probably overkill to try to replace `asctime` with something else.

After I've seen the results, I think the format should include `module`.  Putting it along with `funcname` (`%(funcName)s.%(funcName)s`) would be good.

Reply:
I wonder why logging doesn't offer a way to include the class name.  It seems that `module` will be the name of the file without `.py`, but I didn't see anything else in http://docs.python.org/3/library/logging.html#logrecord-attributes that mentions class.

Reply:
It looks like the class name is not available directly. So you'd either have to do some work to inspect the stack which would have a performance penalty or else use a Mixin that saves the class name [like this example. ](https://stackoverflow.com/a/35177483/3708872)

I don't think it's super important here since the funcName is here. 

Also [interesting history about the logging module](https://stackoverflow.com/a/22993896/3708872). It was originally based on Java (Log4j) and wasn't in the standard library originally. 

Also I think it's interesting that we almost always do
`logger = logging.getLogger(__name__)` which should actually be `LOGGER` since it's a often a global.

Reply:
Yeah, I've wondered about why we (I) don't use `LOGGER`. I think I was just following what I'd seen before in T&L code; prior to starting here, I hadn't used `logging` that much.

Reply:
@lsloan, do you really want me to add `module`? Aren't we already getting the full import path in `name`, including the module? I agree it would be nice to have `class` be supported.

Reply:
I suppose when it's `__main__`, we don't see `run_jobs.py`, but in most cases, it'll show `mivideo.mivideo_extract`, or `course_inventory.inventory`. 

We could consider replacing `name` (which is really the logger name) with `module`, but we might lose a little information.

Reply:
If you think `module` is sufficiently represented as part of `name`, then I guess it's not necessary.  Without trying it myself, I'm not totally certain how much difference there is between the two variables.

Reply:
WRT class name, it's probably not worth worrying about.  Instead, we should ensure that multiple classes in the same module don't contain methods with the same names.

Reply:
Okay, I'll check to make sure `module` would truly be redundant. If not, I'll merge.

Reply:
Besides the case of `__main__` (where it would be nice to see `run_jobs`), I think the current configuration is better, as it includes the packages as well (I'm not seeing a super clear way otherwise to do this with the available attributes). I'm going to merge as is.